imageroot/actions/get-configuration/validate-output.json

```diff
@@ -1,10 +1,64 @@
 {
     "$schema": "http://json-schema.org/draft-07/schema#",
-    "title": "get-configuration output",
-    "$id": "http://schema.nethserver.org/passbolt/get-configuration-output.json",
+    "title": "Get passbolt configuration",
+    "$id": "http://nethserver.org/json-schema/task/output/piler/get-configuration",
     "description": "Get passbolt configuration",
     "examples": [
-        {"host": "passbolt.gs.nethserver.net", "lets_encrypt": false}
+        {
+            "host": "passbolt.nethserver.org",
+            "lets_encrypt": true,
+            "admin_url": "https://passbolt.nethserver.org/setup/start/e78b7682-632b-4e2d-86c4-779a0f14edf6/86502cf4-2c99-453b-82fb-e57719da9bbe",
+            "admin_email": "admin@nethserver.org",
+            "admin_created": false,
+            "admin_not_active": false
+        }
     ],
-    "type": "object"
+    "type": "object",
+    "required": [
+        "host",
+        "lets_encrypt",
+        "admin_url",
+        "admin_email",
+        "admin_created",
+        "admin_not_active"
+    ],
+    "properties": {
+        "host": {
+            "description": "Host name for the application, like 'piler.nethserver.org'",
+            "format": "idn-hostname",
+            "title":"Hostname of Passbolt User Interface"
+        },
+        "lets_encrypt": {
+            "type": "boolean",
+            "title": "Let's Encrypt certificate",
+            "description": "Request a valid Let's Encrypt certificate."
+        },
+        "admin_url": {
+            "oneOf": [
+                {
+                    "type": "string",
+                    "description": "URL for creating the admin",
+                    "title": "Admin URL"
+                },
+                {
+                    "const": null
+                }
+            ]
+        },
```

Comment:
no tested but I believe that string can be either empty, none or any value

Reply:
I think it's null when initially unset or it has the admin URL so it's never an empty string like ""
For a string I think also the empty string "" is allowed, am I wrong?

Reply:
yes "" is allowed for a string, but I was (quite) sure that None or null too

Sometimes I can say wrong :D

Reply:
No, without allowing null it throws an error on the settings page because the adminurl is unset (set to None).

```
Validation errors: [admin_url: Must validate one and only one schema (oneOf) admin_url: Invalid type. Expected: string, given: null]
```